core/services/aws/tests/user/cognitoTrigger/lambdas/postConfirmationTrigger.test.ts

```diff
@@ -0,0 +1,76 @@
+import postConfirmationLambda from '../../../../user/cognitoTrigger/lambdas/postConfirmationTrigger'
+import { PostConfirmationTriggerEvent } from 'aws-lambda'
+import { UserService } from '@application/userWorkflows/UserService'
+import DynamoDbTableOperations from '../../../../commons/ddb/DynamoDbTableOperations'
+import { UserDTO } from '@commons/dto/UserDTO'
+
+jest.mock('@application/userWorkflows/UserService')
+jest.mock('../../../../commons/ddb/DynamoDbTableOperations')
+
+describe('postConfirmationLambda Tests', () => {
+  let mockEvent: PostConfirmationTriggerEvent
+  let mockDynamoDbTableOperations: jest.Mocked<DynamoDbTableOperations<UserDTO, any, any>>
+
+  beforeEach(() => {
+    mockEvent = {
```

Comment:
maybe move these to a canned file so that other tests can make use of it

Reply:
Moved mockEvent to a separate file.

You will find with this [Commit](https://github.com/CraftsmenLtd/BloodConnect/pull/113/commits/6d856e5322c15066df61036318febceea7de3529)